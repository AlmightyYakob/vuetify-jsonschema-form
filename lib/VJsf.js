// same as VJsjNoDeps but requiring each used component of vuetify to try working well with vuetify-loader

import VJsfNoNeps from './VJsfNoDeps'

import {
  VAvatar,
  VAutocomplete,
  VBtn,
  VCard,
  VCardActions,
  VCardText,
  VCardTitle,
  VCheckbox,
  VChip,
  VCombobox,
  VContainer,
  VDatePicker,
  VDialog,
  VExpansionPanels,
  VExpansionPanel,
  VExpansionPanelHeader,
  VExpansionPanelContent,
  VFlex,
  VIcon,
  VInput,
  VLayout,
  VListItemContent,
  VListItemTitle,
  VListTileContent,
  VListTileTitle,
  VMenu,
  VSelect,
  VSpacer,
  VSubheader,
  VTabs,
  VTab,
  VTabItem,
  VTextarea,
  VTextField,
  VTimePicker,
  VTooltip
} from 'vuetify/lib'

export default {
  name: 'VJsf',
  components: {
    VAvatar,
    VAutocomplete,
    VBtn,
    VCard,
    VCardActions,
    VCardText,
    VCardTitle,
    VCheckbox,
    VChip,
    VCombobox,
    VContainer,
    VDatePicker,
    VDialog,
    VExpansionPanels,
    VExpansionPanel,
    VExpansionPanelHeader,
    VExpansionPanelContent,
    VFlex,
    VIcon,
    VInput,
    VLayout,
    VListItemContent,
    VListItemTitle,
    VListTileContent,
    VListTileTitle,
    VMenu,
    VSelect,
    VSpacer,
    VSubheader,
    VTabs,
    VTab,
    VTabItem,
    VTextarea,
    VTextField,
    VTimePicker,
    VTooltip
  },
  mixins: [VJsfNoNeps]
}
