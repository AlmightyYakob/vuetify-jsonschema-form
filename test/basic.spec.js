import { mount } from '@vue/test-utils'
import VJsf from '../lib/index.vue'

describe('Main component', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(VJsf)
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.element).toMatchSnapshot()
  })
})
