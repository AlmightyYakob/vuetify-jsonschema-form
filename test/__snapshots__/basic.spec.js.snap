// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component is a Vue instance 1`] = `<!---->`;
